src/components/20-molecules/datepicker/index.scss

```diff
@@ -36,6 +36,12 @@ axa-datepicker {
       border-color: $color-shy-tomato;
       outline-color: $color-shy-tomato;
     }
+    .m-datepicker__input-wrap:hover {
+      .m-datepicker__input-button {
```

Comment:
does not make sense to write two different selectors, but fine for me.

better: `.m-datepicker__input-wrap:hover .m-datepicker__input-button {`

as long as you don't need nesting because of other selectors.